builtin_interfaces/Time stamp
uint16[] id_list

float64[] pwm_percent
float64[] current_ma
float64[] velocity_deg_s
float64[] position_deg
float64[] profile_vel_deg_s
float64[] profile_acc_deg_ss